test/functional/helpers.lua

```diff
@@ -353,6 +365,11 @@ local function clear(...)
         env[#env + 1] = k .. '=' .. v
       end
     end
+    -- if we want to load the vimrc, then we need to regenerate the default
+    -- args just for this invocation
+    if opts.allow_vimrc then
```

Comment:
on current master, can just use `clear{args_rm={'-u'}}` instead. Then `get_default_argv()` is not needed. master also has `